membership.go

```diff
@@ -402,14 +403,17 @@ func multicastAnnounce(addr string) error {
 		logError(err)
 		return err
 	}
-
+	laddr, err := net.ResolveUDPAddr("udp", fmt.Sprintf("%s:0", GetListenIP().String()))
+	if err != nil {
+		logError(err)
+		return err
+	}
 	for {
-		c, err := net.DialUDP("udp", nil, address)
+		c, err := net.DialUDP("udp", laddr, address)
```

Comment:
With the latest commit in place this change may not be needed, although it does not seem wrong either.

Reply:
I agree: it's generally right to specify behaviors.